rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html

```diff
@@ -0,0 +1,131 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags wagtailsettings_tags static %}
+{% get_settings %}
+
+{% block body_class %}
+    app--editorial-event-landing
+{% endblock %}
+
+{% block content %}
+
+    <div class="page page--no-hero">
+
+        <header class="page__header bg bg--dark">
+            <div class="title-area title-area--landing-page grid">
+                <div class="title-area__content">
+                    {% if page.title %}
```

Comment:
should this wrap the entirety of the header or is it still needed even if there's no page title for visual reasons?

Reply:
yep good plan!